### PR TITLE
#163357032 A user should be able to provide a comment

### PR DESCRIPTION
**Pull request by khwilo**
#### What does this PR do?
Adds functionality for a user to create a comment

#### Description of Task to be completed?
- [x] Create a class for a user comment
- [x] Provide a method to search for comment by an author

#### What are the relevant pivotal tracker stories?

[#163357032](https://www.pivotaltracker.com/story/show/163357032)

### CODE REVIEW

**Comment by Ogutu-Brian** [comment]
You can rename the add comment to save

File: comments/comment.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+class Comment:
+    '''Defintions for the comment entity'''
+    def __init__(self, author, message):
+        self.comment_id = len(COMMENTS) + 1
+        self.author = author
+        self.message = message
+        self.created_at = str(datetime.now())
+
+    def add_comment(self, comment):
```
Comment (denniswanjiru) [inline comments/comment.py]: I don't think it's appropriate to get the comment directly from the view. You should probably create the comment from the instance of this class. As follows.
```
def add_comment(self):
         comment = {
            "comment_id": self.comment_id,
            "author": self.author,
            "message": self.message,
            "created_at": self.created_at
         }

         COMMENTS.append(comment)
```

**Comment by khwilo** [comment]
> You can rename the add comment to save

Already done.

**Review by denniswanjiru — APPROVED** [review]

File: comments/comment.py (hunk 1 above)
Comment (khwilo) [reply comments/comment.py]: I've now implemented the feature.

**Pull request merged**